modules/client-configuration/pages/autoinst-distributions.adoc

```diff
@@ -0,0 +1,100 @@
+[[autoinst-distributions]]
+= Autoinstallation Distributions
+
+The autoinstallation process relies on an several files to initiate the installation.
+These files include the Linux kernel, an initial RAM disk, and other files required to boot the operating system in installation mode.
+
+You can extract the needed files from a DVD image.
+For information, see xref:client-configuration:autoinst-distributions.adoc#based-on-iso-image[Distribution Based on an ISO Image].
+
+Alternatively, you can install [package]``tftpboot-installation`` package.
+For information, see xref:client-configuration:autoinst-distributions.adoc#based-on-rpm-package[Distribution Based on a RPM Package].
+
+You must also have a base channel synchronized on your {productname} Server for the same operating system version as those files.
+
+When you have the files ready, and the base channel synchronized, you need to declare the distribution.
+This operation associates the installation files to the base channel.
+The distribution can be referred to by one or more installation profiles.
+For information, see xref:client-configuration:autoinst-distributions.adoc#declare-distribution[Declare an Autoinstallable Distribution].
+
+
+[[based-on-iso-image]]
+== Distribution Based on an ISO Image
+
+This method assumes you have installation media for the operating system you want to install on the clients. 
+This is usually a DVD [path]``.iso`` image that contains the Linux kernel, an [path]``initrd`` file, and other files required to boot the operating system in installation mode.
+
+
+
+.Procedure: Importing Files from Installation Media
+. Copy the installation media to your {productname} Server.
+  For {suse} operating systems, you can download installation media from https://www.suse.com/download/.
+. Loop-mount the ISO image, and copy its contents somewhere:
```

Comment:
The "copy" method seemed better to me because:
* you may have a real CD instead of an ISO image, and you probably don't want to leave it in the CD-ROM reader forever
* it's faster to directly access files than through a loop-mount
* you can remove useless things from your copy to save disk space, or fix things
* you don't have to mess up `/etc/fstab`

and as you said:
* sometimes it happens that the mount during boot does not work or after a migration - SUMA 3.2 to 4.0 - the ISO images do not exist on disk
* the admin may want to free some space and remove ISO images from disk and he do not know why they need it 

Let's keep the current text. We know it's possible to do loop-mounts in `/etc/fstab`, but let's not advertise that method.